migrations/0006_auto_20200227_1051.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-02-27 10:51
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submission', '0042_auto_20200227_1051'),
```

Comment:
I don't think this migration exists

Reply:
It does on my feature-plugins branch.

https://github.com/BirkbeckCTP/janeway/blob/feature-plugins/src/submission/migrations/0042_auto_20200227_1051.py